app/src/lib/drag-and-drop-manager.ts

```diff
@@ -9,6 +9,8 @@ import { Disposable, Emitter } from 'event-kit'
  * drag event.
  */
 export class DragAndDropManager {
+  public isDragInProgress: boolean = false
```

Comment:
If this is public, that means it can also be changed from outside, right?

Maybe we should have a private mutable property, and a public getter?

Reply:
done